### DISCUSSION TITLE
Please add one more font for CJK characters.

### DISCUSSION THREAD

**Discussion by sherlcok314159** [jocmp/capyreader#654] [Feature Requests]
Hi! First of all, thanks for your contributions to Capyreader, and I love it. The default font can be pretty fine for English characters. As for CJK characters such as Chinese, Japanese, etc, there is very popular and open-sourced font in github. https://github.com/lxgw/LxgwWenKai. It has one specific font type for screen reading. Please add [this font](https://github.com/lxgw/LxgwWenKai-Screen/releases/download/v1.501/LXGWWenKaiScreen.ttf).

Really appreciate your help!

![image](https://github.com/user-attachments/assets/dc07e3ee-b02d-4dc5-9d8e-c95d0ccc2479)

**Comment by jocmp** [comment]
Hey @sherlcok314159 thanks for the suggestion! I'll see what I can do.

**Reply by sherlcok314159** [reply]
Thanks! Happy New Year!

**Comment by sherlcok314159** [comment]
Is there anything i can help with? This font is open-sourced and beautiful for screen reading. :-)

**Reply by jocmp** [reply]
Hey @sherlcok314159 sorry for the radio silence. I looked into the font but it would add 18MB to the APK size. It's still on my list but not prioritized.

My goal with a feature like this would be to allow optional fonts to be downloaded. I don't want to complicate the UI by allowing any `.ttf` to be added.

**Reply by sherlcok314159** [reply]
Well. Your concern is reasonable. This font has woff2 [version](https://cdn.jsdelivr.net/gh/sherlcok314159/Files_@main/LXGWWenKaiMono-Regular.woff2) which is only 7 MB. 

On the other hand, letting the user decide which font to download is an awesome feature.
